### DISCUSSION TITLE
Show color-bg-inset text in top of home page

### DISCUSSION THREAD

**Discussion by M97Chahboun** [community/community#54867] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

here some issue on [GitHub home page ](https://github.com)

![Screenshot 2023-05-08 at 16 09 43](https://user-images.githubusercontent.com/69054810/236861445-4f37adf7-155e-4b1e-9998-2574bf1e660b.png)

**Comment by nrsimonelli** [comment]
I am seeing the same text

**Comment by ulatov** [comment]
has already disappeared

**Reply by nrsimonelli** [reply]
🥳

**Comment by Ndamix-Store** [comment]
Bug
